src/movie/Edit.js

```diff
@@ -0,0 +1,183 @@
+import React, { useReducer, useState } from 'react';
+import { useParams, useHistory } from 'react-router';
+import { Link } from 'react-router-dom';
+import { Alert } from 'react-bootstrap';
+import { movieList, genres } from './Const';
+
+import './Edit.scss';
+
+export function EditMovie() {
+  const [formErrorShow, setFormErrorShow] = useState(false);
+  const [submitSuccessShow, setSubmitSuccessShow] = useState(false);
+  const history = useHistory();
+
+  let { id } = useParams();
+  id = Number(id) || 0;
```

Comment:
Since I am passing id in query string hence it will be evaluated to `string` data type. And I had mock data with id as number therefore in order to compare it using `===` I was converting it to numeric data type.